### PR TITLE
Update API docs

### PR DESCRIPTION
**Pull request by baileymatthewr**
Changes from John and Jeff required a few minor updates. Mostly related to measurements on the product-related stuff and endpoints changed for billing-related stuff.

No associated task

### CODE REVIEW

File: _product/create_item_supplier.md [hunk 1]
```diff
@@ -275,30 +275,42 @@ height
...
 width
-: (number) The Width of the SKU in inches
+: (number) The Width of the SKU in "dimension_units"
 
 height
-: (number) The Height of the SKU in inches
+: (number) The Height of the SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/create_item_supplier.md]: should specify a default if none given

File: _product/create_item_supplier.md [hunk 2]
```diff
@@ -275,30 +275,42 @@ height
...
 
 #### Package Measurements Object:
 
 weight
-: (number) The Weight of the packaged SKU in pounds (lbs.)
+: (number) The Weight of the packaged SKU in "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/create_item_supplier.md]: should specify a default if none given

File: _product/create_item_supplier.md [hunk 3]
```diff
@@ -275,30 +275,42 @@ height
...
 width
-: (number) The Width of the packaged SKU in inches
+: (number) The Width of the packaged SKU in "dimension_units"
 
 height
-: (number) The Height of the packaged SKU in inches
+: (number) The Height of the packaged SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/create_item_supplier.md]: should specify a default if none given

File: _product/create_item_supplier.md [hunk 4]
```diff
@@ -275,30 +275,42 @@ height
 #### SKU Measurements Object:
 
 weight
-: (number) The Weight of the SKU in pounds (lbs.)
+: (number) The Weight of the SKU in the "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/create_item_supplier.md]: should specify a default if none given

File: _product/create_sku_supplier.md [hunk 5]
```diff
@@ -202,30 +202,42 @@ height
 #### SKU Measurements Object:
 
 weight
-: (number) The Weight of the SKU in pounds (lbs.)
+: (number) The Weight of the SKU in the "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/create_sku_supplier.md]: should specify a default if none given

File: _product/create_sku_supplier.md [hunk 6]
```diff
@@ -202,30 +202,42 @@ height
...
 width
-: (number) The Width of the SKU in inches
+: (number) The Width of the SKU in "dimension_units"
 
 height
-: (number) The Height of the SKU in inches
+: (number) The Height of the SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/create_sku_supplier.md]: should specify a default if none given

File: _product/create_sku_supplier.md [hunk 7]
```diff
@@ -202,30 +202,42 @@ height
...
 
 #### Package Measurements Object:
 
 weight
-: (number) The Weight of the packaged SKU in pounds (lbs.)
+: (number) The Weight of the packaged SKU in "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/create_sku_supplier.md]: should specify a default if none given

File: _product/create_sku_supplier.md [hunk 8]
```diff
@@ -202,30 +202,42 @@ height
...
 width
-: (number) The Width of the packaged SKU in inches
+: (number) The Width of the packaged SKU in "dimension_units"
 
 height
-: (number) The Height of the packaged SKU in inches
+: (number) The Height of the packaged SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/create_sku_supplier.md]: should specify a default if none given

File: _product/get_item_detail_retailer.md [hunk 9]
```diff
@@ -435,30 +443,42 @@ height
 #### SKU Measurements Object:
 
 weight
-: (number) The Weight of the SKU in pounds (lbs.)
+: (number) The Weight of the SKU in the "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/get_item_detail_retailer.md]: should specify a default if none given

File: _product/get_item_detail_retailer.md [hunk 10]
```diff
@@ -435,30 +443,42 @@ height
...
 width
-: (number) The Width of the SKU in inches
+: (number) The Width of the SKU in "dimension_units"
 
 height
-: (number) The Height of the SKU in inches
+: (number) The Height of the SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/get_item_detail_retailer.md]: should specify a default if none given

File: _product/get_item_detail_retailer.md [hunk 11]
```diff
@@ -435,30 +443,42 @@ height
...
 
 #### Package Measurements Object:
 
 weight
-: (number) The Weight of the packaged SKU in pounds (lbs.)
+: (number) The Weight of the packaged SKU in "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/get_item_detail_retailer.md]: should specify a default if none given

File: _product/get_item_detail_retailer.md [hunk 12]
```diff
@@ -435,30 +443,42 @@ height
...
 width
-: (number) The Width of the packaged SKU in inches
+: (number) The Width of the packaged SKU in "dimension_units"
 
 height
-: (number) The Height of the packaged SKU in inches
+: (number) The Height of the packaged SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/get_item_detail_retailer.md]: should specify a default if none given

File: _product/get_item_list_retailer.md [hunk 13]
```diff
@@ -804,30 +747,42 @@ height
 #### SKU Measurements Object:
 
 weight
-: (number) The Weight of the SKU in pounds (lbs.)
+: (number) The Weight of the SKU in the "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/get_item_list_retailer.md]: should specify a default if none given

File: _product/get_item_list_retailer.md [hunk 14]
```diff
@@ -804,30 +747,42 @@ height
...
 width
-: (number) The Width of the SKU in inches
+: (number) The Width of the SKU in "dimension_units"
 
 height
-: (number) The Height of the SKU in inches
+: (number) The Height of the SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/get_item_list_retailer.md]: should specify a default if none given

File: _product/get_item_list_retailer.md [hunk 15]
```diff
@@ -804,30 +747,42 @@ height
...
 
 #### Package Measurements Object:
 
 weight
-: (number) The Weight of the packaged SKU in pounds (lbs.)
+: (number) The Weight of the packaged SKU in the "weight_units"
+
+weight_units
+: (string) The units utilized by the supplier for weight ('g', 'kg', 'lb', and 'oz' are potential options)
```
Comment (jtprince) [inline _product/get_item_list_retailer.md]: should specify a default if none given

File: _product/get_item_list_retailer.md [hunk 16]
```diff
@@ -804,30 +747,42 @@ height
...
 width
-: (number) The Width of the packaged SKU in inches
+: (number) The Width of the packaged SKU in "dimension_units"
 
 height
-: (number) The Height of the packaged SKU in inches
+: (number) The Height of the packaged SKU in "dimension_units"
+
+dimension_units
+: (string) The units utilized by the supplier for dimensions ('cm', 'm', 'in', and 'ft' are potential options)
```
Comment (jtprince) [inline _product/get_item_list_retailer.md]: should specify a default if none given

[and all other similar ones]

**Comment by baileymatthewr** [comment]
If you two (@jtprince && @jfischerlaunch) can review this again, that would be great.

File: _product/sku_autocomplete_retailer.md [hunk 17]
```diff
@@ -113,41 +123,33 @@ SKU Autocomplete allows you to search for a SKU without having the complete sku_
...
-
-product_images
-: (list) The Product Images list parameter contains the SKU Product Image objects
-
 sku_id
 : (string) The Stock Keeping Unit (SKU) Identifier for the SKU as provided by the Supplier
 
-uuid
-: (string) The Universal Unique Identifier for the SKU
+owner_uuids
```
Comment (jtprince) [inline _product/sku_autocomplete_retailer.md]: @baileymatthewr - we should probably remove owner_uuids (it should be `owner_uuid`, so you or me maybe messed up on the plural).  I need to remove that from the output.

**Comment by baileymatthewr** [comment]
@jtprince && @jfischerlaunch : Unfortunately, I had to make one slight change in order to avoid a conflict on release, depending upon which PR gets released first. The "Add slight change from Edje" commit message you'll see above is where this could have overwritten good data with old bad data.

**Pull request merged**